src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,136 @@
+
+You no longer need to use different tools to work on your application in Okteto and you won't need to worry about rogue external resources continuing to run once you are done with your work.
+
+When you configure External Resources in Okteto you can see those resources within the Okteto UI, can click to go directly to those resources in their respective platform, and include them in your Okteto `deploy` and `destroy` configurations.
+
+## Overview
+
+When you configure the `external` section in your [Okteto manifest](/reference/manifest), keep in mind the following:
+
+- You can create an image that includes your desired external (to Okteto) resources you'd like to use for your application
+- This image can be included in your Okteto manifest's `deploy` and `destroy` sections
+- External resources are deleted when your Okteto namespace is deleted
+- External resources are covered by Okteto's garbage collection process
+
+## Configuring the `external` section
+
+Let's take a look at the composition of the `external` section in the Okteto manifest starting with the high-level structure of this configuration block. There are some important details you need to know before using this feature.
+
+Here's an example of how to define your `external` resources in the Okteto manifest, which you can see in our [References section](/reference/manifest/#external-object-optional):
+
+```yaml
+external:
+  db:
+    notes: docs/database.md
+    icon: database
+    endpoints:
+    - name: db
+      url: https://localhost:3306
+  functions:
+    notes: docs/lambdas.md
+    icon: function
+    endpoints:
+    - name: data-aggregator
+      url: https://fake-id.lambda-url.us-east-1.on.aws.aggregator
+    - name: data-processor
+      url: https://fake-id.lambda-url.us-east-1.on.aws.processor
+```
+
+> You can also use the [dynamic endpoints](/reference/manifest/#use-dynamic-endpoints-for-an-external-url) feature here to provide more flexibility in the URLs.
+
+Every keyword that you see here is opinionated except for `db:` and `functions:`. Those two keywords, the names of the external resources being defined here, can be anything you want them to be. For example, if we want to call the `db:` resource `dundie_award:` you can do so. Similarly, you could name `functions:` something like `parkour:` and it will work just fine.
+
+Whatever name you use to label these resources will be what's shown in the Okteto UI. Choose wisely!
+
+## Defining Your External Resources
+
+As you configure your external resources in this section, keep in mind the following definitions:
+
+`notes:` is used to provide context in the form of a relative link to a markdown file. The contents are displayed in the Okteto UI (see below).
```

Comment:
The link must point to a file that lives on the repository. Also, we have some rules on the markdown we support. should we put them here?

Reply:
@rberrelleza Yes, I think we should include those rules. Are they documented somewhere that you know of?